### ISSUE TITLE
Login page broken in App (Android)

### ISSUE THREAD

**Issue by Jorf88** [IlluminusLimited/CoreUI#63]
Attempting to login on the app results in basically being stuck at the login screen.
Device: Galaxy S10
Android 9 

![Screenshot_20190519-135759_Pinster](https://user-images.githubusercontent.com/50806111/57986092-6609f380-7a3e-11e9-9ace-a338be5311cd.jpg)

Once I get to the login screen, I'm actually presented with two different options to execute whatever path it's trying to take:

![Screenshot_20190519-121201_Android System](https://user-images.githubusercontent.com/50806111/57986136-052eeb00-7a3f-11e9-9811-bcd61ff9b128.jpg)

One option will result in just going straight back to the login screen (no selection of email/facebook or whatever for authentication comes up, it is just straight back to the screen as posted in the first screenshot).

Selecting the second option will:
The first time I do it it will send me back to the home browsing screen like I just opened the app.
Subsequent attempts will pop me up an error that "Pinster keeps stopping".

![Screenshot_20190519-121510_Pinster](https://user-images.githubusercontent.com/50806111/57986115-c7ca5d80-7a3e-11e9-9b00-b62c9f5419b4.jpg)

I have tried:
Clearing Pinster's cache + data.
...
Clearing Expo's cache + data, and Pinster's cache + data.
...
Clearing Expo's cache + data, Pinster's cache + data, uninstalling BOTH of those apps, reinstalling ONLY Pinster. Same behavior.

**Comment by Jorf88** [comment]
I fixed this.
I had to go into my Chrome --> privacy --> clear EVERYTHING. Cookies, cache, history. Then I went to the app settings and cleared any residual cache data.

After that it redirected me via the login screen correctly.
I was still given the two different options (as in screenshot 2).
Choosing the left-most one would result in successful login.
Choosing the right-side option (What Andrew believes is a residual piece of data that exists from me testing an early beta of this from Expo), resulted in it breaking itself as indicated in this thread.
